From 2078c07d38ee7841b89ac8d81afc5e8bfcd4f703 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sun, 8 Oct 2017 00:58:58 +0200
Subject: [PATCH] Include <fcntl.h> to get loff_t definition

Patch borrowed from Alpine Linux at
http://git.alpinelinux.org/cgit/aports/plain/main/parted/fix-includes.patch. Solves
build with musl.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 libparted/fs/xfs/platform_defs.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libparted/fs/xfs/platform_defs.h b/libparted/fs/xfs/platform_defs.h
index 2b55752..17023dc 100644
--- a/libparted/fs/xfs/platform_defs.h
+++ b/libparted/fs/xfs/platform_defs.h
@@ -35,6 +35,7 @@
 #define __XFS_PLATFORM_DEFS_H__
 
 #include <stdio.h>
+#include <fcntl.h>
 #include <stdarg.h>
 #include <assert.h>
 #include <endian.h>
-- 
2.14.2

